build-docker.sh

```diff
@@ -30,26 +39,34 @@ for BITCOIN_ONLY in 0 1; do
   DIRSUFFIX=${BITCOIN_ONLY/1/-bitcoinonly}
   DIRSUFFIX=${DIRSUFFIX/0/}
 
+  SCRIPT_NAME=".build_core_$BITCOIN_ONLY.sh"
+  cat <<EOF > "build/$SCRIPT_NAME"
+    # DO NOT MODIFY!
+    # this file was generated by ${BASH_SOURCE[0]} on $(date)
+    # variant: core build BITCOIN_ONLY=$BITCOIN_ONLY
+    set -e -o pipefail
+    cd /tmp
+    git clone "$REPOSITORY" trezor-firmware
+    cd trezor-firmware/core
+    ln -s /build build
+    git checkout "$TAG"
+    git submodule update --init --recursive
+    poetry install
+    poetry run make clean vendor build_firmware
+    poetry run ../python/tools/firmware-fingerprint.py \
+               -o build/firmware/firmware.bin.fingerprint \
+               build/firmware/firmware.bin
+    chown -R $USER:$GROUP /build
+EOF
+
   docker run -it --rm \
-    -v $(pwd):/local \
-    -v $(pwd)/build/core"${DIRSUFFIX}":/build:z \
+    -v "$DIR:/local" \
+    -v "$DIR/build/core$DIRSUFFIX":/build:z \
     --env BITCOIN_ONLY="$BITCOIN_ONLY" \
     --env PRODUCTION="$PRODUCTION" \
+    --init \
```

Comment:
Why?

Reply:
I believe this is just a recommended best practice for running anything in docker, it won't hurt:
https://github.com/krallin/tini#why-tini

Here is a detailed writeup:
https://github.com/krallin/tini/issues/8

I started using this everywhere after I had issues with signals. E.g. `docker-compose stop` didn't work properly, because the signal wasn't properly propagated to running process inside container.

Reply:
OK, lets' keep this then.